src/ExtraTasksExtension.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace OpenEuropa\CodeReview;
+
+use GrumPHP\Extension\ExtensionInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * Extension that allows to define extra tasks on local grumphp.yml.dist.
+ */
+class ExtraTasksExtension implements ExtensionInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function load(ContainerBuilder $container)
+    {
+        if ($container->hasParameter('extra_tasks')) {
+            $tasks = $container->getParameter('tasks');
+            foreach ($container->getParameter('extra_tasks') as $name => $value) {
+                $tasks[$name] = $value;
+            }
+            $container->setParameter('tasks', $tasks);
```

Comment:
Do you think we should maybe prevent people from adding in the `extra_tasks:` already existing tasks?
The way it is implemented now gives the possibility to actually override all services we are packaging in the Code Review component, which kind of invalidates the whole purpose of standardizing static checks.

We could throw an exception if they try to add an already defined task, we should also cover that with a test. Check[ here](https://phpunit.de/manual/6.5/en/writing-tests-for-phpunit.html#writing-tests-for-phpunit.exceptions.examples.ExceptionTest.php) how to assert exception in PHPUnit.